CHANGELOG.md

```diff
@@ -40,6 +40,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
   - `RPCGRPCRequestMetadata`
   - `RPCGRPCResponseMetadata`
 
+### Fixed
```

Comment:
Wouldn't it be better fitted in "Added" section?

Reply:
Moved to Added